ding/envs/env_manager/subprocess_env_manager.py

```diff
@@ -97,6 +100,7 @@ def __init__(
         """
         super().__init__(env_fn, cfg)
         self._shared_memory = self._cfg.shared_memory
+        self._cuda_shared_memory = self._cfg.cuda_shared_memory if self._shared_memory else False
```

Comment:
maybe we should raise warning if `self._shared_memory=False` but `self._cfg.cuda_shared_memory=True`